src/components/Card/Card.d.ts

```diff
@@ -0,0 +1,16 @@
+export enum Elevations {
+  NONE = '0',
+  S = '1',
+  M = '2',
+}
+
+export enum Sizes {
+  S = 'S',
+  M = 'M',
+}
+
+export interface IProps {
```

Comment:
A few things:

1) is it the standard to name the interfaces/TS stuff as `<file>.d.ts`? It would be good to follow whatever is most common
2) I didn't see this being exported to the main bundle, or through the main component. Maybe I missed it?
3) We should decide from now how we're naming interfaces. I think we can have better names than `I<Props>`, such as simply `Card` or `ICard`

Reply:
`I<Props>` is pretty common to name interfaces in TS, but I prefer the naming style of `CardProps` `CardState` etc..

Reply:
> is it the standard to name the interfaces/TS stuff as <file>.d.ts? It would be good to follow whatever is most common

I think so, I recall seeing this into multiple projects

> I didn't see this being exported to the main bundle, or through the main component. Maybe I missed it?

The build is actually failing because of `[!] Error: Could not resolve './Card.d' from src/components/Card/Card.tsx`, but I don't understand why :cry:

> I<Props> is pretty common to name interfaces in TS, but I prefer the naming style of CardProps CardState etc...

Looks good to me, should we do the same for enums? `CardElevations`, `CardSizes`?

> Will the types be exported into a single d.ts file in the end?

Once the build is fixed, we can created a `index.d.ts` file exporting every types

Reply:
+1 to naming things Card<Something>

Reply:
Renaming ✅ 
I was able to fix the build by moving the type definitions into the component file, but IDK if that's a good practice. WDYT?

Reply:
🆙 @obedparla @tamarasaurus

Reply:
Types are correctly exported, let's keep types into the component file